genesyscloud/architect_datatable/resource_genesyscloud_architect_datatable.go

```diff
@@ -115,19 +115,19 @@ func readArchitectDatatable(ctx context.Context, d *schema.ResourceData, meta in
 			return retry.NonRetryableError(fmt.Errorf("Failed to read architect_datatable %s: %s", d.Id(), getErr))
 		}
 		cc := consistency_checker.NewConsistencyCheck(ctx, d, meta, ResourceArchitectDatatable())
-		d.Set("name", *datatable.Name)
-		d.Set("division_id", *datatable.Division.Id)
+		_ = d.Set("name", *datatable.Name)
+		_ = d.Set("division_id", *datatable.Division.Id)
 
 		if datatable.Description != nil {
-			d.Set("description", *datatable.Description)
+			_ = d.Set("description", *datatable.Description)
```

Comment:
Explicitly showed I was not going to use the error code. This makes it easier for devs to realize an error code does get returning, but we are specifically not doing anything with it.